src/algo/camera-age/l500/camera-age.h

```diff
@@ -4,33 +4,36 @@
 #pragma once
 
 #include <string>
+#include <ctime>
 
 
 namespace librealsense {
 namespace algo {
 namespace camera_age {
 namespace l500 {
 
-class manufacture_time
+class work_week
```

Comment:
What do you think about splitting work_week into utilities/time?

Reply:
Then, maybe we don't need camera-age?

Reply:
You mean move everything from these files to a different location?